fe1-web/parts/lao/organizer/eventCreation/events/CreateElection.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import {
+  View, Platform, TextInput, ScrollView,
+} from 'react-native';
+import 'react-datepicker/dist/react-datepicker.css';
+import { useNavigation } from '@react-navigation/native';
+
+import STRINGS from 'res/strings';
+import DatePicker from 'components/DatePicker';
+import ParagraphBlock from 'components/ParagraphBlock';
+import WideButtonView from 'components/WideButtonView';
+import { Timestamp } from 'model/objects';
+import TextBlock from 'components/TextBlock';
+import DropdownSelector from 'components/DropdownSelector';
+
+function dateToTimestamp(date: Date): Timestamp {
+  return new Timestamp(Math.floor(date.getTime() / 1000));
+}
+
+/**
+ * UI to create a Election Event
+ *
+ * TODO Send the Election event to the organization server when the confirm button is pressed
+ */
+const CreateElection = ({ route }: any) => {
+  const styles = route.params;
+  const navigation = useNavigation();
+  const initialStartDate = new Date();
+  const initialEndDate = new Date();
+  // Sets initial end date to 1 hour later than start date
+  initialEndDate.setHours(initialEndDate.getHours() + 1);
+
+  const [startDate, setStartDate] = useState(dateToTimestamp(initialStartDate));
+  const [endDate, setEndDate] = useState(dateToTimestamp(initialEndDate));
+
+  const [electionName, setElectionName] = useState('');
+  const [electionQuestion, setElectionQuestion] = useState('');
+  const [electionBallots, setElectionBallots] = useState(['']);
+  const [electionBallotCounter, setElectionBallotCounter] = useState(2);
+  const votingMethods = [STRINGS.election_method_Plurality];
+  const [votingMethod, setVotingMethod] = useState(votingMethods[0]);
+  const ballotOptionsUIComponents = [];
+
+  const buildDatePickerWeb = () => {
```

Comment:
To my knowledge, the [React DatePicker library](https://reactdatepicker.com/) only supports the Web. 
I made a native Component `DatePicker.native.tsx` on another branch but haven't been able to test it properly since I can't get the project to run on my iPhone yet. 
The issues I get are when the required file is outside the folder fe1-web (such as the ones in the protocol folder), then it can't find it and can't compile. 
I was planning on asking about this in our meeting tomorrow.

Reply:
This might take a fair amount of time to figure out, and I'd be happy to work with you on it, as it's something we're definitely interested in and the engineering team did not have time to look into proper react native support.

We can either meet earlier (3pm onwards) or stay longer in the meeting, after covering @CaiMusso 's needs, in case he wants to leave earlier. Let me know on Slack.

Reply:
Marked as resolved bacause we decided to leave the native date-picker for future work. It will not be in this PR.